src/main/java/io/quarkus/search/app/hibernate/AnalysisConfigurer.java

```diff
@@ -158,6 +173,16 @@ void configureJapanese(ElasticsearchAnalysisConfigurationContext context) {
                         "icu_normalizer",
                         "html_strip");
 
+        context.analyzer(suggestionAnalyzer(language)).custom()
+                .tokenizer("kuromoji_tokenizer")
+                .tokenFilters(
+                        // To make all words in lowercase.
+                        "lowercase",
+                        // To convert characters into ascii ones, e.g. à to a or ę to e etc.
+                        "asciifolding",
```

Comment:
I'm not sure converting Japanese to Ascii will go well...

Reply:
that's true 🙈 😃. I suspect that it does not affect characters it doesn't know .... since we have this filter just above in the default analyzer (it is the last one in the list)